pypeit/reduce.py

```diff
@@ -172,19 +172,17 @@ def extract(self, global_sky, sobjs_obj):
             for iobj in range(self.sobjs.nobj):
                 sobj = self.sobjs[iobj]
                 plate_scale = self.get_platescale(sobj)
-                '''
-                if self.pypeline == 'Echelle':
-                    # Grab the positive object only
-                    thisobj = (self.sobjs.ech_orderindx == iord) & (
-                            self.sobjs.ech_objid > 0)  # pos indices of objects for this slit
-                    sobj = self.sobjs[np.where(thisobj)[0][0]]
-                    # Plate scale
-                    plate_scale = self.spectrograph.order_platescale(sobj.ECH_ORDER,
-                                                                 binning=self.binning)[0]
-                else:
-                    sobj = self.sobjs[iord]
-                    plate_scale = self.spectrograph.detector[self.det - 1]['platescale']
-                '''
+#                if self.pypeline == 'Echelle':
```

Comment:
Looks like this logic was moved to the get_platescale function so we can delete this commented out text.

Reply:
Done.